server/game/cards/10-TEL/AkodoMotivator.js

```diff
@@ -0,0 +1,27 @@
+const _ = require('underscore');
+const DrawCard = require('../../drawcard');
+const AbilityDsl = require('../../abilitydsl');
+
+class AkodoMotivator extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            title: 'Opponent discards an equal number of cards at random',
+            when: {
+                onCardsDiscardedFromHand: (event, context) => {
+                    const discardedFromOwnHand = (event.player === context.player);
+                    const discardedByOpponentsEffect = (event.player === event.context.player.opponent);
```

Comment:
I would prefer this to be `event.player.opponent === event.context.player`
`event.player` always exists but `event.context.player` doesn't always exist

Reply:
Done.